libraries/engage/core/helpers/googleMaps.js

```diff
@@ -0,0 +1,15 @@
+const BASE_URL = 'https://www.google.com/maps';
+
+/**
+ * Generates a universal link to google maps that shows the direction to given address.
+ * @param {Object} address The address object.
+ * @param {string} address.street Street name and number
+ * @param {string} address.city City name
+ * @param {string} address.postalCode Postal code.
+ * @returns {string}
+ */
+export const generateGoogleMapsDirectionsUrl = (address) => {
```

Comment:
The `address` arg should have a default `{}` to have null-safe property access (NPE)

Reply:
agree, changed